website/docs/plugins/signing.html.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: "registry"
+page_title: "Plugin Signing"
+sidebar_current: "docs-plugins-signing"
+description: |-
+  Terraform plugin signing trust levels
+---
+
+# Plugin Signing
+
+There are four levels of signing and programattic authentication for plugins in Terraform:
```

Comment:
"Levels of signing" doesn't really capture what's going on here for me. Alternative lede:

> Terraform providers installed from the Registry are cryptographically signed, and the signature is verified at time of installation. There are three types of provider signatures, each with different trust implications: